src/hotspot/share/gc/serial/serialHeap.cpp

```diff
@@ -762,6 +762,14 @@ bool SerialHeap::no_allocs_since_save_marks() {
          _old_gen->no_allocs_since_save_marks();
 }
 
+void SerialHeap::scan_evacuated_objs(YoungGenScanClosure* young_cl,
+                                     OldGenScanClosure* old_cl) {
+  do {
+    oop_since_save_marks_iterate(young_cl, old_cl);
```

Comment:
This method can probably be inlined; I don't see any other usages.

Reply:
Inlined just now.

Reply:
I inlined both `oop_since_save_marks_iterate` and `scan_evacuated_objs`. But it seems that you only want to inline `oop_since_save_marks_iterate`?